pipelinewise/fastsync/commons/target_snowflake.py

```diff
@@ -60,12 +73,13 @@ def open_connection(self):
                                            autocommit=True,
                                            session_parameters={
                                                # Quoted identifiers should be case sensitive
-                                               'QUOTED_IDENTIFIERS_IGNORE_CASE': 'FALSE'
+                                               'QUOTED_IDENTIFIERS_IGNORE_CASE': 'FALSE',
+                                               'QUERY_TAG': self.create_query_tag(query_tag_props)
```

Comment:
Looking at the screenshots you shared in the PR description, the schema property is just the schema name only.
Do you think that maybe it would be problematic to have not have the DB included? what if we have multiple pipelines loading to the schemas with same name but in different DBs?

I think having either the full qualified schema name or DB also in the query tag would be useful if better filtering and analysing. We can get the db in this scope from `self.connection_config['dbname']`.
wdyt?

Reply:
Voila.

Added to snowflake fastsync by 5672fb0409d24ef201729c9963872ffb2d118b3f
PPW passing the new template with database to target-snowflake by db01e64f2fd2d5da5f2f0451ea013bc3fdc44349
Target-snowflake now understands `{database}` token: https://github.com/transferwise/pipelinewise-target-snowflake/pull/117/files